book/src/7.developer-tools/1.cli/README.md

```diff
@@ -1 +1,63 @@
-tbd
+# Teams CLI
+
+The Teams CLI was created with the intent of supporting developers by making common actions simple to implement with just a command line. The CLI overarching features are:
+
+1. `new` - Create a new Teams AI v2 agent by choosing a template that will be ready to run with one command line.
+1. `config` - Add Teams Toolkit configuration files to your existing Teams AI v2 agent project.
+1. `environment` - Manage multiple environments (e.g. dev, prod) and their keys for your agent.
+
+> [!TIP]
+> With the CLI installed, you can enter `teams <token-arguments> --help` at any command level to access information about the command, tokens, or required arguments.
+
+## Installation
+
+Install the Teams CLI globally using npm:
+
+```sh
+npm install -g @microsoft/teams.cli@latest
+```
+
+> [!TIP]
+> If you prefer not to install globally, all commands below can replace `teams` with npx:
+> > ```npx @microsoft/teams.cli@latest <token-and-arguments>```
```

Comment:
i know technically `token-arguments` is the right term, but consider using just `arguments`. `token` may be confusing